gateway/modules/eventing/helpers.go

```diff
@@ -179,6 +179,22 @@ func (m *Module) generateProcessedEventRequest(eventID string) *model.UpdateRequ
 	}
 }
 
+func (m *Module) generateDLQEventRequest(eventDoc *model.EventDocument) error {
+	req := model.QueueEventRequest{
+		Type: "dlq_event_trigger_" + eventDoc.RuleName,
+		Payload: map[string]interface{}{
+			"id":   eventDoc.ID,
+			"body": eventDoc.Payload,
+		},
+	}
+
+	_, err := m.QueueEvent(context.Background(), m.project, "", &req)
```

Comment:
use the ctx passed above. & use m.adminman > getinternalaccesstoken function to generate token